doc/sphinx/docs/python/tutorials/evolving_a_population.rst

```diff
@@ -0,0 +1,79 @@
+.. py_tutorial_evolving_population
+
+Evolving a population
+=====================
+
+Solving an optimization problem using an optimization algorithm is, in pygmo,
+described as *evolving* a population. In the scientific literature, an interesting
+discussion has developed over the past decades on whether evolution is or not a form of
+optimization. In pygmo we take the opposite standpoint and we regard optimization,
+of all types, as a form of evolution. Regardless on whether you will be using an SQP,
+an interior point optimizer or an evolutionary startegy solver, in pygmo you will
+always have to call a method called ``evolve`` to improve over your initial solutions,
```

Comment:
yes some links are missing as I am waiting for merge of your work .... I guess you did not touch algo ...